### ISSUE TITLE
Tray Icon Color not updating on Windows 10 Pro

### ISSUE THREAD

**Issue by codermonkey55** [containers/podman-desktop#2622]
### Bug description

As shown in the screenshot below, I've changed the "Tray Icon Color" preference to light but it remains unchanged in the taskbar. Additionally, the "(Requires restart)" note in parentheses is ambiguous and does not explicitly state *what* needs to be restarted - I've tried restarted the podman  machine, the podman desktop, I've even restarted my windows machine, all while have the preference set to "Light" and it still remains dark. 

I'm certain at this point that this is a bug.

![image](https://github.com/containers/podman-desktop/assets/17110177/3aaaa966-a28c-4a37-97fb-6586ef53fce2)

### Operating system

Windows 10 Pro 22H2

### Version

1.0.x

### Steps to reproduce

- Install Podman Desktop with defaults. 
- Proceed to the Preferences menu and change the "Tray Icon Color" to the opposite of whatever the current tray icon color is.
- Observe that the Tray Icon color in the taskbar does not update.

### Additional context

Edition	Windows 10 Pro
Version	22H2
Installed on	‎5/‎14/‎2023
OS build	19045.2965
Experience	Windows Feature Experience Pack 1000.19041.1000.0

**Comment by benoitf** [comment]
thanks for the report @codermonkey55 

the restart should be like: right click on the tray icon and then exit, then relaunch Podman Desktop (but you did reboot your computer so it should have worked as well)

I'll need to check with a developer on Windows

**Comment by jeffmaury** [comment]
Tested on Win10 and don't see a change after Podman Desktop restart

**Comment by jeffmaury** [comment]
> Tested on Win10 and don't see a change after Podman Desktop restart

After several tests it seems related to the Windows theme: if you select the light theme then the tray icon is updated accordingly with the setting.

**Comment by lstocchi** [comment]
I didn't test on Win10 but i'm quite sure it is the same behavior.
I think @codermonkey55 was expecting a different color when switching to light. The light one is actually black so there are no problem with the setting update. I tried and the icon gets updated
@mairin should we switch the icons?
 
Dark
![image](https://github.com/containers/podman-desktop/assets/49404737/ad4d7a28-74f5-467c-9c60-3a98ca5e1091)

Light
![image](https://github.com/containers/podman-desktop/assets/49404737/845f2547-d8ac-4678-b82a-30ddf6e14c7b)

**Comment by mairin** [comment]
Sure I can do.

**Comment by curd0** [comment]
The issue is still reproducible in the latest **v.1.5.2**

![image](https://github.com/containers/podman-desktop/assets/12750951/ce9a8716-e05e-4d6b-84bd-4abb34c017a1)

Edition Windows 11 Enterprise
Version 22H2
Installed on ‎6/‎9/‎2023
OS build 22621.2428
Experience Windows Feature Experience Pack 1000.22674.1000.0

**Comment by dgolovin** [comment]
This is happening to me too, but only if I use 'Custom' theme in windows settings.

**Comment by curd0** [comment]
> This is happening to me too, but only if I use 'Custom' theme in windows settings.

Yes, the same here. I use 'Custom' theme: light Apps, dark Windows

**Comment by ciros88** [comment]
happens on manjaro kde (breath dark theme) too: tray icon color set both to light or dark is always dark (also after having restarted podman desktop)

**Comment by garcipat** [comment]
Still an issue. Color doesnt change when configuring it in v 1.16.1.

**Comment by U52284** [comment]
@deboer-tim can we reopen this or do I have to create a new Ticket and reference this one?

**Comment by vancura** [comment]
This should be fixed by #16081.
